app/javascript/components/dashboard/dashboard-table/component.spec.js

```diff
@@ -0,0 +1,38 @@
+import { mountedComponent, screen } from "test-utils";
+import { fromJS } from "immutable";
+
+import DashboardTable from "./component";
+
+describe("<DashboardTable />", () => {
+  const props = {
+    columns: [],
+    data: [],
+    query: [],
+    title: "testTitle",
+    pathname: "/cases"
+  };
+
+  const state = fromJS({
+    user: {
+      permissions: {
+        cases: ["manage"]
+      }
+    }
+  });
+
+  beforeEach(() => {
+    mountedComponent(<DashboardTable {...props} />, state);
+  });
+
+  it("renders a MUIDataTable />", () => {
+    expect(screen.getByRole("toolbar")).toBeInTheDocument();
+  });
+
+  it("should render text", () => {
+    expect(screen.queryAllByText("testTitle")).toBeTruthy();
+  });
+
+  it("should have attribute aria-label", () => {
```

Comment:
Remove this test. This seems to need usage of checking for props which we can't do.